src/SFA.DAS.FAT.Web.UnitTests/Models/ShortlistViewModelTests/WhenGettingIsOneTable.cs

```diff
@@ -7,6 +7,7 @@ namespace SFA.DAS.FAT.Web.UnitTests.Models.ShortlistViewModelTests
 {
     public class WhenGettingIsOneTable
     {
+        // THESE TESTS ARE COMPLETE SHIT - RANDOM DATA - AMAZING IT WORKS!
```

Comment:
Do you really want to leave this comment in?

Reply:
Err - No!

Reply:
Removed